pkg/storaged/fsys-panel.jsx

```diff
@@ -139,8 +142,7 @@ export class FilesystemsPanel extends React.Component {
                         { title: "Stratis" },
                         { title: mount },
                         {
-                            title: <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])),
-                                Number(pool.TotalPhysicalSize)]}
+                            title: <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])), pool_total]}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18831-20230522-125636-1ec6e298-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)